tests/GameServiceTest.php

```diff
@@ -10,29 +10,44 @@ public function testTiltToRightShouldMovePuckToRightBorder(){
         // GIVEN
         $gameService = new GameService();
         $board = $gameService->newGame();
-        $this->assertEquals($board->getCellType(3,3),Board::BLACK);
+        $this->assertEquals($board->getCellType(3,3)[Board::COLOR_KEY],Board::BLACK);
         // WHEN
         $board = $gameService->tilt($board,Board::EAST);
 
         // THEN
-        $this->assertEquals($board->getCellType(3,3),Board::RED);
-        $this->assertEquals($board->getCellType(4,3),Board::BLACK);
-        $this->assertEquals($board->getCellType(5,3),Board::RED);
+        $this->assertEquals($board->getCellType(3,3)[Board::COLOR_KEY],Board::RED);
+        $this->assertEquals($board->getCellType(4,3)[Board::COLOR_KEY],Board::BLACK);
+        $this->assertEquals($board->getCellType(5,3)[Board::COLOR_KEY],Board::RED);
 
     }
 
     public function testTiltToRightShouldMovePuckToLeftBorder(){
         // GIVEN
         $gameService = new GameService();
         $board = $gameService->newGame();
-        $this->assertEquals($board->getCellType(3,3),Board::BLACK);
+        $this->assertEquals($board->getCellType(3,3)[Board::COLOR_KEY],Board::BLACK);
         // WHEN
         $board = $gameService->tilt($board,Board::WEST);
 
         // THEN
-        $this->assertEquals($board->getCellType(1,3),Board::RED);
-        $this->assertEquals($board->getCellType(2,3),Board::BLACK);
-        $this->assertEquals($board->getCellType(3,3),Board::RED);
+        $this->assertEquals($board->getCellType(1,3)[Board::COLOR_KEY],Board::RED);
+        $this->assertEquals($board->getCellType(2,3)[Board::COLOR_KEY],Board::BLACK);
+        $this->assertEquals($board->getCellType(3,3)[Board::COLOR_KEY],Board::RED);
+
+    }
+
+    public function test_tilt_nwnwns_should_not_end_with_error(){
```

Comment:
why using snake_case now ? And what is `nwnwns` ?

Reply:
I find it more readable than CamelCase, for my test method name I find it more relevant.
And I Agree with you, it's not clear, I test the bug case we had on the precedent sprint, it's a succession of direction. I'll make it more obvious.